src/identity.d.ts

```diff
@@ -1,7 +1,7 @@
 /**
- * Provides Identity functionalty to a web page
+ * Provides Identity functionality to a web page
  */
-export class Identity {
+export class Identity extends EventEmitter {
```

Comment:
I just noticed that it's the wrong EventEmitter because you use `tiny-emitter` package.